README.md

```diff
@@ -16,75 +16,107 @@ Installation
+suite_id = 3
+```
 
-	[API]
-	url = https://yoururl.testrail.net/
-	email = user@email.com
-	password = password
+Or
 
-	[TESTRUN]
-	assignedto_id = 1
-	project_id = 1
-	suite_id = 1
+* Set command line options (see below)
 
 Usage
 -----
-	py.test --testrail=<settings file>.cfg
-
-This will create a test run in TestRail, add all marked tests to run.
-Once the all tests are finished they will be updated in TestRail.
-
-	--tr-testrun-name='My Test Run'
-
-Testruns can be named using the above flag, if this is not set a generated one will be used.
-' Automation Run "timestamp" '
-
-	--no-ssl-cert-check
-
-This flag can be used prevent checking for a valid SSL certificate on TestRail host.
-
-Available flags:
-
-    --testrail            Create and update testruns with TestRail
-    --tr-config=TR_CONFIG
-                          Path to the config file containing information about
-                          the TestRail server (defaults to testrail.cfg)
-    --tr-url=TR_URL       TestRail address you use to access TestRail with your
-                          web browser (config file: url in API section)
-    --tr-email=TR_EMAIL   Email for the account on the TestRail server (config
-                          file: email in API section)
-    --tr-password=TR_PASSWORD
-                          Password for the account on the TestRail server
-                          (config file: password in API section)
-    --tr-testrun-assignedto-id=TR_TESTRUN_ASSIGNEDTO_ID
-                          ID of the user assigned to the test run (config file:
-                          assignedto_id in TESTRUN section)
-    --tr-testrun-project-id=TR_TESTRUN_PROJECT_ID
-                          ID of the project the test run is in (config file:
-                          project_id in TESTRUN section)
-    --tr-testrun-suite-id=TR_TESTRUN_SUITE_ID
-                          ID of the test suite containing the test cases (config
-                          file: suite_id in TESTRUN section)
-    --tr-testrun-name=TR_TESTRUN_NAME
-                          Name given to testrun, that appears in TestRail
-                          (config file: name in TESTRUN section)
-    --tr-no-ssl-cert-check
-                          Do not check for valid SSL certificate on TestRail
-                          host
+
+This will create a testrun in TestRail, add all marked tests to run.
+Once the all tests are finished they will be updated in TestRail:
+
+```bash
+py.test --testrail --tr-config=<settings file>.cfg
+
+or
+
+py.test --testrail --tr-url=https://yoururl.testrail.net/ --tr-email=user@email.com --tr-password=uiqefhqliefl34534
+--tr-testrun-assignedto-id=1 --tr-testrun-project-id=2 --tr-testrun-suite-id=3
+```
+
+This will update testrun in TestRail with ID 1234:
+
+	py.test --testrail --tr-config=<settings file>.cfg --tr-run-id=1234
+
+This will update testplan in TestRail with ID 5678:
+
+	py.test --testrail --tr-config=<settings file>.cfg --tr-plan-id=5678
+	
```

Comment:
Don't think we need all these examples:

py.test --testrail --tr-config=<settings file>.cfg
and
the command line options description will be fine